IssueTracker/IssueTrackerAPI/Controllers/CommentController.cs

```diff
@@ -37,17 +37,31 @@ public async Task<IResult> GetById(int id)
         var comment = await _commentData.GetByIdAsync(id);
         if (comment == null) return Results.NotFound("Couldn't find any comment");
         var commentResponse = _mapper.Map<CommentResponse>(comment);
-        commentResponse.Attachements = await AutoMapperConfig.GetAttachements(comment.Attachements);
-        return Results.Ok(commentResponse);
+        try
+        {
+            commentResponse.Attachements = await AutoMapperConfig.GetAttachements(comment.Attachements);
+            return Results.Ok(commentResponse);
+        }
+        catch (FileSystemException ex)
+        {
+            return Results.Conflict(ex.Message);
```

Comment:
Conflict should only be used for add/edit operations.

Reply:
ok, I'll change